basket/base/templates/admin/dsar-info.html

```diff
@@ -0,0 +1,183 @@
+{% extends "admin/base_site.html" %}
+
+{% block extrastyle %}
+  <style>
+  .content-container {
+    width: 100%;
+    max-width: 1600px;
+  }
+  .form-container {
+    max-width: 33%;
+  }
+  .form-container form input {
+    width: 100%;
+  }
+  .output-container {
+    max-width: 67%;
+  }
+  .output-header {
+    align-items: center;
+  }
+  .button-cpy {
+    border: none;
+    border-radius: 5px;
+    padding: 10px 15px;
+    cursor: pointer;
+    align-items: center;
+    float: right;
+  }
+  .button-cpy span {
+    margin-left: 5px;
+  }
+  .output-text {
+    background-color: var(--body-bg);
+    color: var(--body-fg);
+    padding: 5px 6px;
+    border-radius: 4px;
+    border: 1px solid var(--border-color);
+    margin: 0 0 5px 0;
+    min-height: calc(1.5em * 9);
+  }
+  </style>
+{% endblock %}
+
+{% block content %}
+<div id="content-main">
+
+  <div class="content-container">
+    <div class="form-container">
+      {% if dsar_form.non_field_errors %}
+        {% for error in dsar_form.non_field_errors %}
+          <p class="errornote">
+            {{ error }}
+          </p>
+        {% endfor %}
+      {% endif %}
+
+      <form method="post">
+        {% csrf_token %}
+        <h2>{{ dsar_form.email.label_tag }}</h2>
+        {% if dsar_form.email.errors %}
+          <div class="errornote">{{ dsar_form.email.errors }}</div>
+        {% endif %}
+        <div>{{ dsar_form.email }}</div>
+        <div class="submit-row">
+          <input type="submit" value="Fetch">
+        </div>
+      </form>
+    </div>
+
+    <div class="output-container">
+      {% if dsar_contact %}
+        <h2>User Info:</h2>
+        <table>
+            <tr>
+                <th>Primary Email</th>
+                <td>{{ dsar_contact.email.primary_email }}</td>
+            </tr>
+            <tr>
+                <th>Basket Token</th>
+                <td><a href="https://www.mozilla.org/newsletter/existing/{{ dsar_contact.email.basket_token }}/" target="_blank" title="Open the user's Email Preferences">{{ dsar_contact.email.basket_token }}</a></td>
```

Comment:
Might be nice to add a constant for the pref center path. Not env var based but just make it easy to find if we ever move it.